demos/python_demos/multi_camera_multi_person_tracking/multi_camera_multi_person_tracking.py

```diff
@@ -141,13 +179,28 @@ def main():
     log.info("Creating Inference Engine")
     ie = IECore()
 
-    person_detector = Detector(ie, args.m_detector, args.t_detector,
-                               args.device, args.cpu_extension,
-                               capture.get_num_sources())
+    if args.detections:
+        person_detector = DetectionsFromFileReader(args.detections, args.t_detector)
+    else:
+        if args.m_segmentation:
+            person_detector = MaskRCNN(ie, args.m_segmentation, args.t_segmentation,
+                                       args.device, args.cpu_extension,
+                                       capture.get_num_sources())
+        else:
+            person_detector = Detector(ie, args.m_detector, args.t_detector,
+                                       args.device, args.cpu_extension,
+                                       capture.get_num_sources())
```

Comment:
Given that `--detection`, `--m_segmentation` and `--m_detector` are apparently alternatives, you should make it so that only one can be specified.

Reply:
Fixed. Added check for number of detectors with error if more than one or nothing was specified.